File: C/Easy/1. Two Sum.c
int* twoSum(int* nums, int numsSize, int target, int* returnSize)
{
    int *solution = (int *)malloc(2 * sizeof(int));
    *returnSize = 2;

    for (int i = 0; i < numsSize; i++) {
        for (int j = i + 1; j < numsSize; j++) {
            if (nums[j] + nums[i] == target) {
                printf("\nSolution has been found ");
                solution[0] = i;
                solution[1] = j;
                return solution;
            }
        }
    }
    
    return 0;
}

File: Rust/Easy/1. Two Sum.rs
impl Solution {
    pub fn two_sum(nums: Vec<i32>, target: i32) -> Vec<i32> {
        
        let mut solution:Vec<i32> = Vec::new();
        let mut i:usize = 0;
        let mut j:usize = 0;

        while i < nums.len() {
            j = i + 1;
            while j < nums.len() {
                let sum = nums[i] + nums[j];
                if sum == target {
                    solution.push(i as i32);
                    solution.push(j as i32);
                    return solution;
                }
                j = j + 1;
            }
            i = i + 1;
        }

        return solution;
    }
}
